foundation/core/variables.tf

```diff
@@ -121,3 +121,51 @@ variable "enable_point_to_site_vpn" {
   description = "If enabled, creates User Point to Site VPN configuration in CAF foundation."
   default     = true
 }
+
+variable "slack_func_identifier" {
+  type    = string
+  default = "billing-alert-func"
+}
+
+variable "enable_budget_alerts" {
+  type        = bool
+  description = "Feature flag to enable billing alerts"
+  default     = false
+}
+
+variable "func_identifier" {
+  type        = string
+  default     = "billing-alert-func"
+  description = "string used to identity billing alert function resources"
+}
+
+variable "slack_webhook_url" {
+  type        = string
+  description = "URL or a Slack webhook to send the Slack alerts to"
+  default     = ""
+}
+
+variable "teams_webhook_url" {
+  type    = string
+  default = ""
+}
+
+variable "budgets" {
+  type    = map(any)
+  default = {}
+}
+
+variable "budget_tags" {
+  type    = map(any)
+  default = {}
+}
+
+variable "sas_time_start" {
+  type    = string
+  default = ""
+}
+
+variable "sas_time_end" {
+  type    = string
+  default = ""
+}
```

Comment:
I assume we don't want this to expire so can we set it to now + a very long amount of time?